server/app/views/errors/InternalServerError.java

```diff
@@ -0,0 +1,82 @@
+package views.errors;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+import static j2html.TagCreator.*;
+
+import com.google.inject.Inject;
+import com.typesafe.config.Config;
+import j2html.tags.specialized.ATag;
+import j2html.tags.specialized.DivTag;
+import j2html.tags.specialized.H1Tag;
+import play.i18n.Messages;
+import play.mvc.Http;
+import play.twirl.api.Content;
+import services.MessageKey;
+import views.BaseHtmlView;
+import views.HtmlBundle;
+import views.LanguageSelector;
+import views.applicant.ApplicantLayout;
+import views.components.LinkElement;
+import views.style.ApplicantStyles;
+import views.style.ErrorStyles;
+
+public final class InternalServerError extends BaseHtmlView {
```

Comment:
**nitpick:** please add javadoc for this class

Reply:
Ok, but if I'm adding it here I'm adding it to the 404 page also ;)

Reply:
thanks!